dbbackup/dbcommands.py

```diff
@@ -282,12 +283,12 @@ def run_command(self, command, stdin=None, stdout=None):
 
     def read_file(self, filepath, stdout):
         """ Read the specified file to stdout. """
-        print("  Reading: %s" % filepath)
+        sys.stdout.write("  Reading: %s" % filepath)
```

Comment:
Probably fine, that part probly won't be mocked anyways.. there's one more below